src/confirmation-window/index.tsx

```diff
@@ -1,9 +1,19 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 import App from "./App";
+import getSdk from "pointsdk/pointsdk/sdk";
+import getEthProvider from "pointsdk/pointsdk/ethProvider";
+import getSolanaProvider from "pointsdk/pointsdk/solanaProvider";
+const version = browser.runtime.getManifest().version;
+const point = getSdk("https://point", version);
```

Comment:
Let's not use this hostname here, because it will give the confirmation window a privileged access to proxy api that it doesn't need.